### ISSUE TITLE
Does gitbook mindmaps not support markdown syntax at present?

### ISSUE THREAD

**Issue by mnlin0905** [aleen42/gitbook-mindmaps#2]
![image](https://user-images.githubusercontent.com/17874282/118356405-b1145e00-b5a7-11eb-850f-ddd46cb65a02.png)

However, when I used it locally, I found that it did not take effect

Is there any way to support this?

**Comment by mnlin0905** [comment]
@aleen42 dear friend

**Comment by mnlin0905** [comment]
https://my.oschina.net/woniu666/blog/4758038

**Comment by aleen42** [comment]
This is a just a plugin to generate markup with the following syntax mentioned in README:

````
```mind
- 1
  - 1.1
    - 1.1.1
  - 1.2
- 2
- 3
```
````

It won't parse the whole page content like what you intend to.

**Comment by aleen42** [comment]
[gitbook-treeview](https://github.com/aleen42/gitbook-treeview) may be something you want.

**Comment by mnlin0905** [comment]
@aleen42  hi

I mean: bold, italics, delete line, etc

Because of some content, I need special tags

Also, sometimes I need to wrap the content, but now I can't

I try to use `<br>`, but I can't achieve the effect of line feed

**Comment by mnlin0905** [comment]
![image](https://user-images.githubusercontent.com/17874282/118448344-78e65a00-b724-11eb-96b1-a78235429c93.png)

Like this rich effect

**Comment by mnlin0905** [comment]
Sometimes I may add a URL link:

```txt
* one 
  * two
    * three,linkto [some tile](some link url)
```

but ` [some tile](some link url)`  But it doesn't take the form of a link

**Comment by aleen42** [comment]
I need to research it and try to implement this.

**Comment by mnlin0905** [comment]
@aleen42 Thank you very much

**Comment by aleen42** [comment]
To implement such a feature, I need to migrate from [dundalek/markmap](https://github.com/dundalek/markmap)  to [gera2ld/markmap](https://github.com/gera2ld/markmap) progressively.

**Comment by aleen42** [comment]
I have decided to reuse [dundalek/markmap](https://github.com/dundalek/markmap) and implement the feature myself. Try it!, I released `gitbook-mindmaps@1.1.4`.

**Comment by mnlin0905** [comment]
> 我决定重用[dundalek/markmap](https://github.com/dundalek/markmap)并自己实现该功能。试试吧！，我发布了`gitbook-mindmaps@1.1.4`。

genius !!!
thanks
